plugins/dicom_viewer/web_client/views/DicomView.js

```diff
@@ -0,0 +1,349 @@
+import vtkImageSlice from 'vtk.js/Sources/Rendering/Core/ImageSlice';
+import vtkImageData from 'vtk.js/Sources/Common/DataModel/ImageData';
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
+import vtkImageMapper from 'vtk.js/Sources/Rendering/Core/ImageMapper';
+import vtkInteractorStyleImage from 'vtk.js/Sources/Interaction/Style/InteractorStyleImage';
+import vtkOpenGLRenderWindow from 'vtk.js/Sources/Rendering/OpenGL/RenderWindow';
+import vtkRenderer from 'vtk.js/Sources/Rendering/Core/Renderer';
+import vtkRenderWindow from 'vtk.js/Sources/Rendering/Core/RenderWindow';
+import vtkRenderWindowInteractor from 'vtk.js/Sources/Rendering/Core/RenderWindowInteractor';
+
+wrap(ItemView, 'render', function (render) {
+    this.once('g:rendered', function () {
+        $('.g-item-header').after('<div id="g-dicom-view"></div>');
+        /* eslint-disable no-new */
+        new DicomView({
+            el: $('#g-dicom-view'),
+            parentView: this,
+            item: this.model
+        });
+        /* eslint-enable no-new */
+    }, this);
+    render.call(this);
+});
+
+var DicomView = View.extend({
+    events: {
+        'click #dicom-zoom-in': function (event) {
+            event.preventDefault();
+            this.camera.zoom(9 / 8);
+            this.iren.render();
+        },
+        'click #dicom-zoom-out': function (event) {
+            event.preventDefault();
+            this.camera.zoom(8 / 9);
+            this.iren.render();
+        },
+        'click #dicom-reset-zoom': function (event) {
+            event.preventDefault();
+            this.autoZoom();
+            this.iren.render();
+        },
+        'click #dicom-first': function (event) {
+            event.preventDefault();
+            this.setIndex(0);
+        },
+        'click #dicom-previous': function (event) {
+            event.preventDefault();
+            this.previous();
+        },
+        'click #dicom-next': function (event) {
+            event.preventDefault();
+            this.next();
+        },
+        'click #dicom-last': function (event) {
+            event.preventDefault();
+            this.setIndex(this.files.length - 1);
+        },
+        'click #dicom-play': function (event) {
+            event.preventDefault();
+            this.play();
+        },
+        'click #dicom-pause': function (event) {
+            event.preventDefault();
+            this.pause();
+        },
+        'click #dicom-auto-levels': function (event) {
+            event.preventDefault();
+            this.autoLevels();
+            this.iren.render();
+        },
+        'input #dicom-slider': _.debounce(function (event) {
+            this.setIndex(event.target.value);
+        }, 10)
+    },
+
+    initialize: function (settings) {
+        this.item = settings.item;
+        this.files = [];
+        this.index = 0;
+        this.first = true;
+        this.playing = false;
+        this.playRate = 500;
```

Comment:
Seems like this property (and maybe others?) should be able to be passed in via settings to override the default value.